### PR TITLE
IoT Analytics Workshop update

### PR DESCRIPTION
**Pull request by dhwalters423**
*Description of changes:*

- Significant reformatting
- Align MQTT topic space to best practices
- Address issues raised in https://github.com/aws-samples/aws-iot-analytics-workshop/issues/2
- Added new feature: Customer managed S3 buckets
- Explanations of several steps and added recap sections to summarise the workshop at certain points.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -295,13 +293,13 @@ In this section we will create the IOT Analytics components, analyze data and de
...
 
 #### Create Container Data Set
 
 A container data set allows you to automatically run your analysis tools and generate results. It brings together a SQL data set as input, a Docker container with your analysis tools and needed library files, input and output variables, and an optional schedule trigger. The input and output variables tell the executable image where to get the data and store the results.
 
-Navigate to AWS IOT Analytics console, in the left navigation pane, choose Analyze.
+Navigate to AWS IoT Analytics console, in the left navigation pane, choose Analyze.
 
      
     1. Click on Analyze -> Data Sets → Create
```
Comment (neelmitra) [inline README.md]: We donot have the Analyze options anymore on the home page for IoT Analytics , can you please update it to -> 
Click on Data sets ->  Create

The same changes are required for Datastore, pipeline, channel etc.

**Review by neelmitra — COMMENTED** [review]
Thanks , i have reviewed looks good, other than 1 comment, where all the Analyze -> needs to be removed and replaced by respective components. Since the dashboard looks different now. Thanks.

File: README.md (hunk 1 above)
Comment (dhwalters423) [reply README.md]: Hi Neel, this should already be done - can you look at commit 6f7dfc8 which is the latest.

Comment (neelmitra) [reply README.md]: ok looks good then, feel free to merge with the main branch. Thanks for your help.

**Pull request merged**